Doc/whatsnew/3.7.rst

```diff
@@ -97,6 +97,14 @@ New Modules
 Improved Modules
 ================
 
+locale
+------
+
+Added another argument *monetary* in :meth:`format_string` of :mod:`locale`.
+If *monetary* is true, the conversion uses monetary thousands separator and
+grouping strings.
+(Contributed by Garvit in :issue:`10379`.)
+
```

Comment:
Small nit: our convention appears to be to put the Contributed by as part of the paragraph when there's only one paragraph.  Ends up formatted the same in html, but might as well make the ReST consistent as well.